android/app/src/main/java/org/fossasia/openevent/utils/DateUtils.java

```diff
@@ -173,4 +173,9 @@ public static String getRelativeTimeFromTimestamp(String timeStamp) throws Parse
                 (timeCreatedDate.getTime()),
                 System.currentTimeMillis(), android.text.format.DateUtils.SECOND_IN_MILLIS);
     }
+
+    public static boolean isUpcomingSession(Date start, Date end, Date current) {
```

Comment:
As a concept of separation of concerns, a class should do only thing and do it well. DateUtils has no context of session and should not contain any method specific to it. Its sole purpose is to handle conversion of ISO8601 format dates